source/welcome/whats-new-in-v72.rst

```diff
@@ -1,7 +1,7 @@
 What’s new in Mattermost Boards v7.2
```

Comment:
I also noticed that the TOC link is currently broken here https://docs.mattermost.com/guides/welcome-to-mattermost.html#mattermost-interface-changes.

Reply:
Thanks Amy! Fixed.